ci/docker/runtime_functions.sh

```diff
@@ -38,9 +38,28 @@ clean_repo() {
 build_jetson() {
     set -ex
     pushd .
-    mv make/crosscompile.jetson.mk make/config.mk
+
+
+    # workaround for old version of ccache that has a bug working with nvcc
+
+    echo '#!/bin/sh\nccache ${CC} "$@"\n' > cc
```

Comment:
Are the slashes different on purpose?

Reply:
this is @lebeg 's part

Reply:
It also seems to fail: http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/incubator-mxnet/detail/PR-11036/2/pipeline